### ISSUE TITLE
sort incorrect

### ISSUE THREAD

**Issue by zzjjzzgggg** [ranger/ranger#1212]
#### Runtime Environment

- Operating system and version: 
- Terminal emulator and version: 
- Python version: 
- Ranger version/commit: 
- Locale: 

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```

```

**Comment by toonn** [comment]
I can confirm this, seems like the setlocal takes precedence.

**Comment by doronbehar** [comment]
Is there a difference between this and #1440? BTW, I really hope these can be fixed so I'll be able to open with `mpv` tracks named with leading zeros in the correct order..

**Comment by toonn** [comment]
Yes, #1440 is a unicode handling issue, while this is about the `setlocal` in terfering with the order set manually.
